### ISSUE TITLE
Tracking implants show duplicates on crew monitorg

### ISSUE THREAD

**Issue by Lamrr** [space-wizards/space-station-14#28335]
## Description

Crew members who are implanted with implant trackers (and have suit sensors on) will appear twice on the crew monitoring console as two separate (but identical) entries. This clutters the monitor with redundant entries

**Reproduction**

1. Turn suit sensors on (any level)
2.  Implant a tracking implant
3.  You will appear on the crew monitoring console twice

**Screenshots**

![Screenshot from 2024-05-28 16-25-28](https://github.com/space-wizards/space-station-14/assets/96937466/7d991d46-78d0-405f-8b16-23f130a7efd2)

**Comment by deltanedas** [comment]
this is because they have suit coords on

**Comment by Lamrr** [comment]
> this is because they have suit coords on

Yes, it's the case with all levels of suit sensors. Really the implant should modify the existing entry and not make another one

**Comment by lzk228** [comment]
intended

**Comment by Lamrr** [comment]
> intended

As I understand, the intention is to have the user tracked by the suit sensor network. But they really should just make the existing entry (if available) to give co-ords, so the entries on the monitor are per-user not per-device. Creating duplicate entries is silly, making things cluttered and confusing.
